pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -268,7 +293,7 @@ func (csh *cisScanHandler) deployApp(appInfo *appInfo) error {
 		"debugWorker":              appInfo.debugWorker,
 		"overrideBenchmarkVersion": appInfo.overrideBenchmarkVersion,
 	}
-	logrus.Debugf("appAnswers: %v", appAnswers)
+	logrus.Debugf("cisScanHandler: deployApp: appAnswers: %v", appAnswers)
```

Comment:
q: is it ok to log answers object here? any data that shouldn't go in logs?

Reply:
there isn't anything sensitive in the answers